src/flicking.js

```diff
@@ -8,31 +8,30 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 
 	// jscs:enable validateLineBreaks, maximumLineLength
 	/**
-	 * To build flickable UI
+	 * Component to build flickable carousel UI
 	 * @group egjs
-	 * @ko 플리킹 UI를 구성한다.
+	 * @ko 캐로셀 플리킹 UI 구성 컴포넌트

```

Comment:
Could you reinforce description? like this.
https://github.com/naver/egjs/blob/master/src/infiniteGrid.js#L210-L212